postgres-persistence/src/main/java/com/netflix/conductor/postgres/config/PostgresConfiguration.java

```diff
@@ -21,33 +21,42 @@
 import com.netflix.conductor.postgres.dao.PostgresQueueDAO;
 import javax.sql.DataSource;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.flyway.FlywayConfigurationCustomizer;
+import org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration;
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.DependsOn;
+import org.springframework.context.annotation.Import;
 
 @SuppressWarnings("SpringJavaInjectionPointsAutowiringInspection")
 @Configuration(proxyBeanMethods = false)
 @EnableConfigurationProperties(PostgresProperties.class)
 @ConditionalOnProperty(name = "conductor.db.type", havingValue = "postgres")
+@Import(DataSourceAutoConfiguration.class)
```

Comment:
A comment here explaining the reason for the inclusion would be helpful. Since its an "auto configuration", it isn't immediately obvious why the Import is necessary.